### DISCUSSION TITLE
In app payment method does not use default account

### DISCUSSION THREAD

**Discussion by jmiezitis** [AppGround-io/bluetooth-keyboard-and-mouse-support#156] [Q&A] [answered]
Recently purchased premium version of bluetooth mouse and keyboard.  The in app payment process does not indicate which account is being used, does not allow you to change it and doesn't use the default account or at least the one that google play is using.
The result of the above now means the app is licensed to my work account,  which may change and not my personal account.  It also created some confusion when it advised me that there wasn't any payment methods setup.  Which almost resulted in me not making the purchase as I thought it may be a scam.
I now have a gift card attached to my work account which probably will not be used again.
However the app works really well for what I wanted i.e. copy text on the android and paste into my laptop.  Fantastic work on that piece.

**Comment by jmiezitis** [comment, marked as answer]
I now know why this happened.  The app was installed from a browser not from google play on the device.  The browser was logged into google using my work account.  The in app purchase account must be tied to the installing account.
This issue can be closed.
Can the license be transferred?

**Reply by AppGround-io** [reply]
The payment and account handling is done by the Play Store and as a developer I don't get any information about the account. So transferring a licence is not possible. You can send me mail and I can give you a code that you can redeem for your account.

**Reply by jmiezitis** [reply]
Thanks for the offer but after looking at this a bit more I don't think it is necessary.
The same app list appears regardless of which account I use.
This is all a non issue and can be closed.
